app/models/article.rb

```diff
@@ -0,0 +1,28 @@
+class Article < ApplicationRecord
+  belongs_to :blog, class_name: 'Blog', foreign_key: :blogs_id
+
+  has_many :article_comments
+  has_many :article_likes
+
+  before_save :update_author_columns
+  after_save :update_counter
+  after_destroy :reduce_counter
+
+  validates :title, presence: true, length: { minimum: 2, maximum: 100 }
+  validates :text, presence: true, length: { minimum: 10 }
```

Comment:
- I think it will be better if we give a maximum of 1000 words so as to regulate it.